src/cluster/src/cli/start/sys.rs

```diff
@@ -0,0 +1,40 @@
+use crate::cli::start::StartOpt;
+use crate::cli::ClusterCliError;
+use crate::sys::{SysConfig, SysInstaller};
+use crate::ClusterError;
+use crate::error::SysInstallError;
+
+pub fn install_sys(opt: StartOpt, upgrade: bool) -> Result<(), ClusterCliError> {
+    install_sys_impl(opt, upgrade).map_err(ClusterError::InstallSys)?;
+    Ok(())
+}
+
+fn install_sys_impl(opt: StartOpt, upgrade: bool) -> Result<(), SysInstallError> {
+    let mut builder = SysConfig::builder();
+    builder.with_namespace(opt.k8_config.namespace);
+
+    match opt.k8_config.chart_location {
```

Comment:
maybe this should be moved to `opt` rather than here

Reply:
Hmm, maybe, but right now this does what we need it to do. I don't see an urgent need to move it, and it's not immediately obvious to me how things would look by trying to rewrite this.

Reply:
sounds good